includes/blocks.theme.inc

```diff
@@ -25,6 +25,47 @@ function contacts_theme_preprocess_block(&$variables) {
     }
   }
 
+  if ($variables['base_plugin_id'] == 'facet_block') {
+    $collapsed = TRUE;
+    if (in_array($variables['plugin_id'], array('facet_block:crm_roles', 'facet_block:crm_type'))) {
+      $collapsed = FALSE;
+    }
+    $facet_id = \Drupal\Component\Utility\Html::getUniqueId('facet-dropdown-button');
+    // Add properties to facet block title.
+    $variables['attributes']['class'][] = 'mb-4';
+    $variables['title_attributes']['class'][] = 'facet-toggle';
+    $variables['title_attributes']['data-target'][] = "#$facet_id";
+    $variables['title_attributes']['data-toggle'][] = 'collapse';
+    $variables['title_attributes']['aria-controls'][] = $facet_id;
+    // Add properties to facet block content wrapper.
+    $variables['content_attributes']['id'] = $facet_id;
+    $variables['content_attributes']['class'][] = 'collapse';
+
+    if ($collapsed) {
```

Comment:
Let's put a comment indicating what were doing, something like `// Set the default collapsed states.`